amigos/amigos/Weather_Reading.py

```diff
@@ -0,0 +1,36 @@
+#Sid Arora

```

Comment:
We should modularize this and integrated with cli if not done already.

Maybe need a separate folder for demo-type scripts like this?

There is merge conflict annotation here still as well.

Reply:
Undertood. I will let ssid know. About the merge problem, I will fix that tomorrow.